### DISCUSSION TITLE
Attempting to access BleakClient.mtu_size triggers warning about _acquire_mtu

### DISCUSSION THREAD

**Discussion by fixermark** [hbldh/bleak#1270] [Q&A] [answered]
* bleak version: 0.20.1
* Python version: 3.9.12
* Operating System: Ubuntu 5.4.0-144-generic
* BlueZ version (`bluetoothctl -v`) in case of Linux: bluetoothctl: 5.53

### Description

After creating a BleakClient and calling `client.connect()`, I attempt to access `client.mtu_size`.

*Expected result*:
mtu_size retrieved.

*Observed result*:
A warning is emitted:

```
<$USER>/miniconda3/lib/python3.9/site-packages/bleak/backends/bluezdbus/client.py:620: UserWarning: Using default MTU value. Call _acquire_mtu() or set _mtu_size first to avoid this warning.        
  warnings.warn(
```

The `mtu_size` shows as 23. Monocle's preferred MTU is 128.

Even though it is single-underscore-prefixed, I also attempted to call `_acquire_mtu` first, but the method does not appear to exist on `BleakClient`. Scanning the source code tells me it is a method on [BluezDBus client](https://github.com/hbldh/bleak/blob/develop/bleak/backends/bluezdbus/client.py#L542) however, but it's unclear to me how to get to that method from BleakClient (and I suspect I'm not supposed to be calling it directly...).

### What I Did

I'm writing a small client for the Brilliant Labs Monocle, which has a MicroPython shell accessible via UART.

Minimum workable example:
``` Python
import asyncio
from bleak import BleakClient

async def example():
  address = '<address of Monocle>'
  client = BleakClient(address)
  await client.connect()
  print(client.mtu_size)

asyncio.run(example())
```
I am unaware of anything Monocle-specific that would prevent reproduction of this phenomenon on other devices (but a Monocle is all I have access to at this time).

### Logs

Logs attached.
[out.log](https://github.com/hbldh/bleak/files/11128397/out.log)

**Comment by dlech** [comment, marked as answer]
This is the documented behavior: https://bleak.readthedocs.io/en/latest/api/client.html#bleak.BleakClient.mtu_size

Please use `BleakGATTCharacteristic.max_write_without_response_size` instead as suggested or provide your own implementation as in https://github.com/hbldh/bleak/blob/develop/examples/mtu_size.py. You can hard-code 128 as the MTU instead of using `_acquire_mtu`  if you are sure it will always be correct.

**Reply by fixermark** [reply]
It appears `max_write_without_response_size` on Monocle's RX characteristic is also returning 20, which may be an issue with Monocle. We'll hard-code a value in our application. Thank you for the help!

**Reply by dlech** [reply]
Actually the 20 can be because BlueZ is older than v5.62. But hard-coding in that case is still the appropriate workaround.
